### PR TITLE
[9.0][ADD] account_invoice_view_payment

### PR DESCRIPTION
**Pull request by JordiBForgeFlow**
Account Invoice View Payment
============================

This module allows users to access directly to the payment from an invoice
when registering a payment or afterwards.

The option to open the payment when it's being registered is useful
when the user needs to do a follow-up step on the payment, such as printing
the associated check.

Usage
=====

Registering a payment
---------------------
#. Go to an open customer invoice or vendor bill and press 'Register Payment'.
#. Enter the payment details and press 'Validate & View Payment'

After payment has been made
---------------------------
#. Go to an open customer invoice or vendor bill and press the link
   'View Payments' that appears next to the list of payments made for that
   invoice.

### CODE REVIEW

**Comment by JordiBForgeFlow** [comment]
cc @Eficent

**Comment by JordiBForgeFlow** [comment]
@gmeficent @mreficent can you review?

**Comment by gmeficent** [comment]
The 'View Payments' link from the paid invoice is broken and the following error appears.

image

The other functionalities work just fine.

**Review by gmeficent — CHANGES_REQUESTED** [review]
The 'View Payments' link from the paid invoice is broken and the following error appears.

image

The other functionalities work just fine.

**Comment by JordiBForgeFlow** [comment]
@lreficent can you review?

**Comment by JordiBForgeFlow** [comment]
@gmeficent thanks, this is now fixed.

File: account_invoice_view_payment/views/account_payment_view.xml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+                <button name="post_and_open_payment" class="oe_highlight"
+                        string="Validate &amp; View Payment" type="object"/>
+            </button>
+        </field>
+    </record>
+
+    <record id="view_account_payment_from_invoices" model="ir.ui.view">
+        <field name="name">account.register.payments.wizard</field>
+        <field name="model">account.register.payments</field>
+                <field name="inherit_id"
```
Comment (LoisRForgeFlow) [inline account_invoice_view_payment/views/account_payment_view.xml]: Fix this indentation

**Review by LoisRForgeFlow — APPROVED** [review]
LGTM :+1: - Just a minor comment

**Review by gmeficent — APPROVED** [review]
Code and functional review, good to go :+1:

**Review by MiquelRForgeFlow — APPROVED** [review]
:+1:

**Pull request merged**